python_modules/libraries/dagstermill/dagstermill/manager.py

```diff
@@ -335,6 +335,13 @@ def yield_result(self, value, output_name="result"):
         # pass output value cross process boundary using io manager
         step_context = self.context._step_context  # pylint: disable=protected-access
         # Note: yield_result currently does not support DynamicOutput
+
+        # dagstermill assets do not support yielding additional results within the notebook:
+        if len(step_context.job_def.asset_layer.asset_keys) > 0:
```

Comment:
this was the best way i could find to determine if the notebook was being run as a op vs as an asset. if there's a more elegant way lmk!